CHANGELOG.md

```diff
@@ -0,0 +1,35 @@
+<!-- markdownlint-disable no-duplicate-heading -->
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+<!-- towncrier release notes start -->
```

Comment:
@abhinavsingh do not remove this line. This is how Towncrier will know where to inject the generated changelog.